Assets/Scripts/CountDownTimer.cs

```diff
@@ -0,0 +1,45 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.SceneManagement;
+
+public class CountDownTimer : MonoBehaviour
+{
+    public float timeValue;
+    public Text texTimer;
```

Comment:
all public variables need to be in PascalCase, also texTimer should be renamed as TextTimer:
`public float TimeValue;
 public Text TextTimer;`